Brocfile.js

```diff
@@ -123,7 +123,11 @@ for (var packageName in packages.dependencies) {
   trees.push(pickedEs6Lib);
 
   // AMD lib
-  var transpiledAmdLib = transpileES6(libTree, { moduleName: true, type: 'amd' });
+  var transpiledAmdLib = new ES6Modules(libTree, {
+    moduleName: true,
+    type: 'amd',
+    strict: true,
```

Comment:
@rwjblue this is still failing, despite passing in `strict: true` (which I assume gets passed to esperanto)

https://travis-ci.org/tildeio/htmlbars/builds/59270937#L145-L157

```
File: dom-helper/build-html-dom
You must be in strict mode (pass `strict: true`) to use named imports or exports
Error: You must be in strict mode (pass `strict: true`) to use named imports or exports
    at Class.toFormat (/home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/node_modules/esperanto/dist/esperanto.js:2532:11)
    at Class.module.exports.CachingWriter.extend._transpileThroughCache (/home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/index.js:230:22)
    at Class.module.exports.CachingWriter.extend._handleFile (/home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/index.js:202:22)
    at Class.<anonymous> (/home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/index.js:185:16)
    at Array.forEach (native)
    at Class.module.exports.CachingWriter.extend._updateEachFile (/home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/index.js:183:8)
    at Class.module.exports.CachingWriter.extend.updateCache (/home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/index.js:141:74)
    at /home/travis/build/tildeio/htmlbars/node_modules/broccoli-es6modules/node_modules/broccoli-caching-writer/index.js:92:34
    at lib$rsvp$$internal$$tryCatch (/home/travis/build/tildeio/htmlbars/node_modules/rsvp/dist/rsvp.js:489:16)
    at lib$rsvp$$internal$$invokeCallback (/home/travis/build/tildeio/htmlbars/node_modules/rsvp/dist/rsvp.js:501:17)
```